include/flatbuffers/reflection.h

```diff
@@ -362,7 +362,6 @@ template<typename T, typename U> class pointer_inside_vector {
         reinterpret_cast<uint8_t *>(flatbuffers::vector_data(vec_)) + offset_);
   }
   T *operator->() const { return operator*(); }
-  void operator=(const pointer_inside_vector &piv);
```

Comment:
I agree this declaration is not needed.
`std::vector<U> &vec_;` is non-copyable and non-movable.

Can you remove a similar declaration in `reflection.cpp`?
https://github.com/google/flatbuffers/blob/c4231c3cb99159978ca5f0669a07ef1aa02b88e3/src/reflection.cpp#L299